standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/TestHiveMetaStoreTimeout.java

```diff
@@ -117,7 +123,7 @@ public void testResetTimeout() throws Exception {
         .build(conf);
     try {
       client.createDatabase(db);
-    } catch (MetaException e) {
+    } catch (Exception e) {
```

Comment:
why is this change?

Reply:
Because we have changed to use the remote MetaStore server in this test class, so the timeout exception was wrapped in `TTransportException` rather than `MetaException`.